webviews/components/timeline.tsx

```diff
@@ -122,7 +122,7 @@ const ReviewEventView = (event: ReviewEvent) => {
 	const comments = groupCommentsByPath(event.comments);
 	const reviewIsPending = event.state.toLocaleUpperCase() === 'PENDING';
 	return (
-		<div id={reviewIsPending ? 'pending-review' : null} className="comment-container comment">
+		<div id={reviewIsPending ? 'pending-review' : undefined} className="comment-container comment">
```

Comment:
drive-by fix to make IntelliSense happy because `id` is of type `string | undefined` and not `string | null`